t/webservice.t

```diff
@@ -81,15 +81,12 @@ transactionally {
 
     my $newUser;
     subtest "create-account" => {
-        my $userData = \(
-            :username('foo@bar.ch'),
-            :firstname('Erika'),
-            :lastname('Mustermann'),
-            :password('myPass'),
-            :organisation('Org'),
-        );
-        ok $newUser = $ws.create-account($user, $userData), "Create new account";
-        is $newUser.username, 'foo@bar.ch', "User has expected username";
+        ok $username = $ws.create-account(
+            $user,
+            'foo@bar.com', 'myPassword', 'myKey',
+            'Erika', 'Mustermann', 'MyOrg'
+        ), "Create new account";
+        is $username, 'foo@bar.com', "User has expected username";
```

Comment:
Also add a test for the case where `Any` is passed for the things that are optional, to cover that case.

Reply:
Added another test, some error handling, and OpenAPI validation.